nncf/common/accuracy_aware_training/training_loop.py

```diff
@@ -230,52 +251,78 @@ def remove_registry_prefix(algo_name):
         raise RuntimeError('No compression algorithm that supports adaptive compression '
                            'accuracy-aware training was specified')
 
-    def run(self, model, train_epoch_fn, validate_fn, configure_optimizers_fn=None,
-            dump_checkpoint_fn=None, tensorboard_writer=None, log_dir=None):
+    def run(self, model: TModel, train_epoch_fn, validate_fn, configure_optimizers_fn=None,
```

Comment:
Type hints

Reply:
It was decided to improve code quality within a separate task